test/v4/controllers/RetrieveCalculationControllerSpec.scala

```diff
@@ -75,7 +78,12 @@ class RetrieveCalculationControllerSpec
           .wrap(responseWithR8b, RetrieveCalculationHateoasData(nino, TaxYear.fromMtd(taxYear), calculationId, responseWithR8b))
           .returns(HateoasWrapper(responseWithR8b, hateoaslinks))
 
-        runOkTest(OK, Some(mtdResponseWithR8BJson))
+        runOkTestWithAudit(
+          expectedStatus = OK,
+          maybeAuditRequestBody = Some(mtdResponseWithR8BJson),
+          maybeExpectedResponseBody = Some(mtdResponseWithR8BJson),
+          maybeAuditResponseBody = None
+        )
```

Comment:
We don't want to audit the request body, we want to audit the response body:
```suggestion
        runOkTestWithAudit(
          expectedStatus = OK,
          maybeExpectedResponseBody = Some(mtdResponseWithR8BJson),
          maybeAuditResponseBody = Some(mtdResponseWithR8BJson)
        )
```

BTW: I think the tests are passing (even though the controller is not set up to audit the request body) because the `event` method at the bottom of this test uses `None` for the request body regardless of what this code here passes for `maybeAuditRequestBody`.